renku/ui/cli/storage.py

```diff
@@ -24,123 +24,86 @@
    :prog: renku storage
    :nested: full
 
-Pulling files from git LFS
```

Comment:
We need to add a note somewhere in the docs that the stuff that used to be at renku storage ... has now moved to renku lfs .... Also you probably want to print a warning in the renku storage command that the LFS stuff has moved to another thing.

And in general, I am not sure how you deprecate things in the CLI. Are you sure you want to just move this without a period when you warn about the change - and give people a chance to adapt?

Reply:
A nice way to tacle this is to leave the stuff related to LFS under storage (maybe not show them in the --help) but then if they are used as renku storage pull then print the deprecation warning. And if they are used as renku lfs pull then we do nothing. But they are available in both places.

Reply:
Ignore this I realize we already do exactly what I wrote. Just saw it later in my review.